bot-samples/GPT3-template/README.md

```diff
@@ -0,0 +1,21 @@
+# Bot Description
+
+This bot queries the user for a historical figure or fictional character, and then uses GPT-3 to generate language, facilitating a conversation with the user. It contains boilerplate code and an integration to get a Conversation Builder bot connected to the GPT-3 API easily.
+
+# Requirements
+
+- A [GPT-3 developer key](https://beta.openai.com/)
+- A conversational cloud account
+
+# Getting Started Steps
+
+1. Import the "GPT3 Character Template" bot from this repo into your Conversational Cloud account
+2. In the bot's "GPT3" integration, add a Credential for GPT-3. This should be an "Access Token" type credential. The access token will be the one GPT-3 developer key provided by OpenAI, and the token type should be "Bearer"
```

Comment:
@cshape Let's include a link to our documentation on creating credentials here for ease of access.